VungleAdapter/src/main/java/com/chartboost/mediation/vungleadapter/VungleAdapter.kt

```diff
@@ -202,47 +161,32 @@ class VungleAdapter : PartnerAdapter {
                 .trim()
                 .takeIf { it.isNotEmpty() }
                 ?.let { appId ->
-                    Vungle.init(appId, context.applicationContext, object : InitCallback {
+                    VungleAds.init(context, appId, object : InitializationListener {
                         override fun onSuccess() {
-                            Plugin.addWrapperInfo(
-                                VungleApiClient.WrapperFramework.vunglehbs,
+                            VungleAds.setIntegrationName(
+                                VungleAds.WrapperFramework.vunglehbs,
                                 adapterVersion
                             )
 
-                            resumeOnce(
-                                Result.success(
-                                    PartnerLogController.log(
-                                        SETUP_SUCCEEDED
-                                    )
-                                )
-                            )
+                            resumeOnce(Result.success(PartnerLogController.log(SETUP_SUCCEEDED)))
                         }
 
-                        override fun onError(exception: VungleException) {
-                            PartnerLogController.log(SETUP_FAILED, "Error: $exception")
-
+                        override fun onError(vungleError: VungleError) {
+                            PartnerLogController.log(SETUP_FAILED, "Error: $vungleError")
```

Comment:
Do we want to map their error to one of ours?

Reply:
Good catch!